HoneyBadgers.ConsoleApp/Repositories/MovieRepository.cs

```diff
@@ -84,5 +85,121 @@ private string StringValidation(int minLength, int maxLength)
 
             return input;
         }
+        public void PrintMovies()
+        {
+            Console.WriteLine("LIST OF MOVIES:");
+            foreach (var movie in Movies)
+            {
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine(
+                    $"TITLE:{movie.Title}\nRELEASE YEAR: {movie.Year}\nDIRECTOR: {movie.Director}\nWRITER: {movie.Writer}\nACTORS: {movie.Actors}\nPLOT: {movie.Plot}\nGENRE: {movie.Genre}\nCOUNTRY: {movie.Country}\nRAITING: {movie.ImdbRating}");
```

Comment:
nie wiem czy to nie będzie za długie jeśli każda dana będzie wyświetlana w nowej linii ale nie mam pomysłu jak to inaczej zrobić

Reply:
Można ewentualnie rozbić to na 2 kolumny

Reply:
poprawione